meta-resin-common/recipes-core/dropbear/dropbear_%.bbappend

```diff
@@ -24,5 +24,6 @@ do_install_append() {
 
     install -d ${D}${sysconfdir}/default
     echo 'DROPBEAR_PORT="22222"' >> ${D}/etc/default/dropbear # Change default dropbear port to 22222
+    echo 'DROPBEAR_RSAKEY_DIR=/var/lib/dropbear' >> ${D}/etc/default/dropber # Change default dropbear host key location
```

Comment:
Wouldn't it be better to have it in etc as bind mount from conf partition? In this way we can avoid bind copy (which we talked in the past that case lead us into troubles).